client/interfaces.go

```diff
@@ -279,6 +282,25 @@ type ContainerSnapshotCopyArgs struct {
 	Live bool
 }
 
+// The ContainerConsoleArgs struct is used to pass additional options during a
+// container console session
+//
+// API extension: console
+type ContainerConsoleArgs struct {
+	// Bidirectional fd to pass to the container
+	Terminal io.ReadWriteCloser
+
+	// Control message handler (window resize)
+	Control func(conn *websocket.Conn)
+}
+
+// The ContainerConsoleLogArgs struct is used to pass additional options during a
+// container console log request
+//
+// API extension: console
+type ContainerConsoleLogArgs struct {
+}
```

Comment:
What's this for? Can we simplify the client API by dropping it and removing the associated arguments on Get and Delete?

Reply:
I actually would like to leave them there in case we ever get a request by users to e.g. clear specific amounts of data from the ringbuffer. This way we don't need to break API. Don't you think this might be worth it?